src/sql/parts/jobManagement/views/jobHistory.component.ts

```diff
@@ -123,6 +123,18 @@ export class JobHistoryComponent extends Disposable implements OnInit {
 			filter: this._treeFilter,
 			renderer: this._treeRenderer
 		}, {verticalScrollMode: ScrollbarVisibility.Visible});
+		let overview: HTMLElement = $('.overview-tab').get(0);
+		let monacoShell: HTMLElement = $('.monaco-shell').get(0);
```

Comment:
should we be changing event handlers at the global shell scope?  It seems like a more localized fix should be possible?